projects/midnight-commander.org/package.yml

```diff
@@ -0,0 +1,42 @@
+distributable:
+  url: http://ftp.midnight-commander.org/mc-{{ version }}.tar.xz
+  strip-components: 1
+
+versions:
+  github: MidnightCommander/mc/tags
+
+provides:
+  - bin/mc
+  - bin/mcdiff
+  - bin/mcedit
+  - bin/mcview
+
+dependencies:
+  invisible-island.net/ncurses: '*'
+  gnome.org/glib: '>=2.30'
+  gnu.org/gettext: '>=0.18.2'
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: '*'
+    tea.xyz/gx/make: '*'
+    gnu.org/autoconf: '>=2.64'
+    gnu.org/automake: '>=1.12'
+    gnu.org/libtool: '*'
+    freedesktop.org/pkg-config: '^0.29'
+  script: |
+    # Needed for the compiler to `#include <term.h>`
+    export CPATH=$CPATH:{{deps.invisible-island.net/ncurses.prefix}}/include/ncursesw
```

Comment:
could be in `env` YAML node. You can put $CPATH in there and it will be assigned after ours.

Reply:
Yum. Better.